### DISCUSSION TITLE
Custom user and MustVerifyEmail

### DISCUSSION THREAD

**Discussion by aliqasemzadeh** [filamentphp/filament#579] [Help] [answered]
Hi
In my app I have `implements MustVerifyEmail` then I can not add ` implements FilamentUser`.
How can I have too contract?

**Comment by ryangjchandler** [comment, marked as answer]
@alighasemzadeh You can implement more than one contract by separating the interface names with a comma:

```php
class User implements MustVerifyEmail, FilamentUser
{
    //
}
```
